src/components/MainMenu/Menu.module.scss

```diff
@@ -23,7 +24,11 @@
 }
 
 .main {
-  display: block;
+  display: inline-block;

```

Comment:
questo cambialo nuovamente in `block`. Vuoi che prenda tutto lo spazio disponibile (aggiungi anche `height:100%`)